tools/wadlto/jaxrs/src/main/java/org/apache/cxf/tools/wadlto/jaxb/CustomizationParser.java

```diff
@@ -118,8 +118,9 @@ private void addBinding(String bindingFile) throws XMLStreamException {
 
         Element root = null;
         try {
-            URIResolver resolver = new URIResolver(bindingFile);
-            root = StaxUtils.read(resolver.getInputStream()).getDocumentElement();
+            try (URIResolver resolver = new URIResolver(bindingFile)) {
```

Comment:
redundant inner try

Reply:
:+1, fixed, thanks!

Reply:
there are same changes in another places...

Reply:
Tried to address all of them, there are some places where I had to keep nested`try` structures since the exiting behavior (throw or return) would change.